store/tikv/region_cache.go

```diff
@@ -1516,7 +1516,7 @@ func (s *Store) reResolve(c *RegionCache) {
 		// we cannot do backoff in reResolve loop but try check other store and wait tick.
 		return
 	}
-	if store == nil {
+	if store == nil || store.State == metapb.StoreState_Tombstone {
```

Comment:
@HunDunDM question, do we need drop region when state be `StoreState_Offline`?

Reply:
`store == nil` means the store is a tombstone. If the store doesn't exist, `pdClient.GetStore()` returns an error: https://github.com/tikv/pd/blob/bc63de897afa69cfcc5828f45431fe3999fed4b3/client/client.go#L1159-L1168